### DISCUSSION TITLE
Same package name causing conflicts?

### DISCUSSION THREAD

**Discussion by protonpony** [LawnchairLauncher/lawnchair#4681] [Q&A] [answered]
I'm an Android enthusiast, but not a developer. Every so often, Lawnchair 14 is mysteriously replaced with Lawnchair 2 by a package manager (fdroid). Upon doing so, the app can no longer be uninstalled or updated because disabling the Device Admin role doesn't always work. This almost always ends in a factory reset and a lot more work for me, despite having taken steps to prevent the app from automatically updating. I noticed the two versions share the same package name. Is this typical? I'd be grateful for any advice.

**Comment by SuperDragonXD** [comment, marked as answer]
This is due to some issues with Lawnchair's `versionCode`, which should be fixed on Lawnchair 14 Beta 3.

I'd suggest disabling auto-updates for *Lawnchair* specifically (probably in Play Store or something similar). If that isn't possible, you can use `Lawnchair 14 Beta 2` which uses a different package name from Lawnchair V2.
